Python/tests/package_management_file_test.py

```diff
@@ -133,7 +136,9 @@ def test_install_targz_files():
         full_package = os.path.join(path_to_packages, package)
         _create(module_name=module, package_file=full_package, class_to_check=class_to_check)
 
-
+# Test a zip that is not a package, then use sqlcmd to make sure it is not in the external_libraries table
+#
+@pytest.mark.skipif(sys.platform.startswith("linux"), reason="Issues with sqlcmd (_execute_sql function) in Linux")

```

Comment:
There is an issue with sqlcmd connecting  (i think because of the driver). The sqlcmd check is actually not necessary since we check using a normal t-sql query already, it was redundant. I just removed this.